webapp/src/styles.scss

```diff
@@ -95,6 +95,9 @@ button {
 	&::-webkit-scrollbar-thumb {
 		background-color: mat-color($accent, 600);
 	}
+
+	//noinspection CssInvalidPropertyValue
+	overflow: overlay;
```

Comment:
Why is this needed? What does it do? Why does it need a special directive to disable the linter? Add comments to code if needed.

Reply:
the comment is webstorm specific and should be ignored by the linter. 
`overlay` is a non standard value only used by webkit based browser that makes the scrollbar not take any space, so the content doesn't get resized when the scrollbar is visible.

I'll add a comment which links to a definition